src/Data/List/Relation/Binary/Disjoint/Propositional/Properties.agda

```diff
@@ -8,18 +8,19 @@
 
 module Data.List.Relation.Binary.Disjoint.Propositional.Properties where
 
-open import Level using (Level)
-open import Function.Base using (_∘_)
-open import Function.Bundles using (_⇔_)
-open import Data.List.Base
-open import Data.List.Relation.Binary.Disjoint.Propositional
-import Data.List.Relation.Unary.Any as Any
-open import Data.List.Relation.Unary.All as All
+
+open import Data.List.Base using (concat; deduplicate; List; []; _∷_)
```

Comment:
```suggestion
open import Data.List.Base using (List; []; _∷_; concat; deduplicate)
```
By convention, types and constructors should always precede functions defined on them!